group_vars/production/vars.yml

```diff
@@ -2,8 +2,8 @@
 
 # Production runs from master branches by default
 
-django_git_branch: master
-celery_git_branch: master
+django_git_branch: fitting
```

Comment:
I'd revert these two lines, as they can get overridden from the Vagrantfile or command line when needed.